tools/packaging/common/envoy_bootstrap.json

```diff
@@ -38,7 +38,7 @@
     "stats_tags": [
       {
         "tag_name": "cluster_name",
-        "regex": "^cluster\\.((.+?(\\..+?\\.svc\\.cluster\\.local)?)\\.)"
+        "regex": "^cluster\\.((.*)\\.)"
```

Comment:
I think this is problematic.  There are a bunch of Envoy metrics that have the form of `cluster.<name>.<other>.<suffix>` (ex: `cluster.<name>.ssl. connection_error`).

I think a _slightly_ more correct regex is something like:

`^cluster\.((.+?(\..+?\.svc\.cluster\.local)|(.+?\.(com|org|net|edu|gov|mil)))?\.)`

But it _might_ be easier to find all of the post-cluster name expressions in Envoy stats and add those as the end delimiter than it would be to list all of the tlds, etc.

Reply:
ah just realized that we have that format. last time quick read and doesn't pay attention.
https://www.envoyproxy.io/docs/envoy/latest/configuration/upstream/cluster_manager/cluster_stats

Reply:
at least there is 1488 TLDs right now
https://data.iana.org/TLD/tlds-alpha-by-domain.txt

Reply:
Any name can be a TLD (ex `svc-name.namespace`) when not dealing with FQDN anyways (or custom DNS servers, etc), so I don't think that is viable.

Reply:
I don't think both of the approach is efficient in form of regex length.

Reply:
the simplest approach is to include all cluster metrics
https://www.envoyproxy.io/docs/envoy/latest/configuration/upstream/cluster_manager/cluster_stats
```
^cluster\.(.*(?=upstream_cx_total|upstream_cx_active|upstream_rq_retry|ssl|tcp_stats))
```

but not sure that this length of regex is supported by Envoy itself.

Reply:
I think that regex is fine. The only issue is somehow keeping it in sync with evolving envoy stats.

@howardjohn i wonder if we could consider escaping `.`s in cluster names ?